.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,33 @@
+### Description
+Please include a brief summary of the change you've made and the issue that it fixes.
+
+Add a link to the relavant story in Jira
+[SASS-434](https://jira.tools.tax.service.gov.uk/browse/SASS-434)
```

Comment:
This Jira link probably shouldn't be included. If you want to template it perhaps you could do something like
[SASS-CHANGEME](ADD URL HERE)